### PR TITLE
Update release process

### PR DESCRIPTION
**Pull request by dinhlongviolin1**

### CODE REVIEW

**Review by FredLL-Avaiga — DISMISSED** [review]

**Review by jrobinAV — DISMISSED** [review]
LGTM. Even if it s a bit strange to have gui dependency required.

**Review by FabienLelaquais — DISMISSED** [review]
That will work for this specific repo and image dir name, which is fine at this point.
I don't see your reluctance though...

**Review by jrobinAV — DISMISSED** [review]

**Review by jrobinAV — DISMISSED** [review]

File: tools/modify_readme.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import re
+import sys
+
+repo_name = sys.argv[1]
+branch_name = sys.argv[2]
+# Regex pattern <img\s+([^>]*?)(?<!['"])(?<!\/)src\s*=\s*(['"])(?!http|\/)(.*?)\2([^>]*?)>
```
Comment (FabienLelaquais) [inline tools/modify_readme.py:6]: Using the r""  syntax would have prevented you to add this comment... and simplify the next line

**Review by FabienLelaquais — APPROVED** [review]
Looking really good!

File: tools/modify_readme.py (hunk 1 above)
Comment (dinhlongviolin1) [reply tools/modify_readme.py:6]: for double quotes you still need the \ so it would almost be the same

**Pull request merged**